Fieldset: Improved SR experience by using `aria-labelledby` for both legend and description (#4029)
navikt/Designsystemet: .changeset/quick-pillows-visit.md

---
"@navikt/ds-react": patch
---

> Fieldset: Use `aria-labelledby` instead of `aria-describedby` for improved screen reader experience

Author: ```suggestion
Fieldset: Improved screen reader experience by using `aria-labelledby` for both legend and description
```
